pages/linux/a2disconf.md

```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> Disable an Apache configuration file on Debian-based OSes.
+> More information: <https://httpd.apache.org/>.
+
+- Disable a configuration file:
+
+`sudo a2disconf {{configuration_file}}`
+
+- Don't show informative messages:
+
+`sudo a2disconf -q {{configuration_file}}`
```

Comment:
Is there a long form of the `-q` option at all? In tldr-pages we prefer to use long options where possible.

Same goes for the other instances of short options.

Reply:
I replaced it with `--quiet`.
`a2query` does not seem to have any long option though.